com.jwestall.Forecast.json

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "com.jwestall.Forecast",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "cosmic-ext-forecast",
+    "finish-args": [
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--talk-name=com.system76.CosmicSettingsDaemon",
+      "--filesystem=xdg-config/cosmic:ro"
+    ],
+    "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "modules": [
+      {
+        "name": "forecast",
+        "buildsystem": "simple",
+        "build-options": {
+            "env": {
+                "CARGO_HOME": "/run/build/forecast/cargo"
+            }
+        },
+        "build-commands": [
+            "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+            "cargo --offline build --release --verbose",
+            "install -Dm0755 ./target/release/cosmic-ext-forecast /app/bin/cosmic-ext-forecast",
+            "install -Dm0644 ./res/com.jwestall.Forecast.desktop /app/share/applications/com.jwestall.Forecast.desktop",
+            "install -Dm0644 ./res/com.jwestall.Forecast.metainfo.xml /app/share/metainfo/com.jwestall.Forecast.metainfo.xml",
+            "install -Dm0644 ./res/icons/hicolor/scalable/apps/com.jwestall.Forecast.svg /app/share/icons/hicolor/scalable/apps/com.jwestall.Forecast.svg"
+        ],
+        "sources": [
+          {
+            "type": "git",
+            "url": "https://github.com/cosmic-utils/forecast.git",
+            "commit": "bb5e4cd75f602475c04c899bd5da12c6d020d10c"
+          },
+	  "cargo-sources.json"
```

Comment:
can you indent this properly ?

Reply:
Would line 43 not be in the same indentation as line 42? (I'll be happy to fix indentation just looking for clarification)

Reply:
No it should start at the `},`, that whole file is another source.

Reply:
I see it now, should be fixed per latest commit.